src/unix/notbsd/mod.rs

```diff
@@ -124,6 +124,68 @@ s! {
         pub machine: [::c_char; 65],
         pub domainname: [::c_char; 65]
     }
+
+    pub struct perf_event_attr {
+        pub type_: u32,  // called `type`
+        pub size: u32,
+        pub config: ::c_ulonglong, // TODO: __u64
```

Comment:
@alexcrichton there is some weirdness here: they are declared as `__u64`, which is a typedef of `unsigned long long` even on x86_64. In libc, we have `type uint64_t = u64` which winds pu as `unsigned long`. This passes on x86_64, but I'm not sure what best to do to make sure it's portable.